srcpkgs/zeroinstall/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'zeroinstall'
+pkgname=zeroinstall
+version=2.16
+revision=1
+wrksrc="0install-v${version}"
+build_style=gnu-makefile
+make_install_target="install_system"
+make_check_args="test"
```

Comment:
you probably want `make_check_target`

Reply:
Νο need, I just removed the line as the makefile runs the test target by default ;)